docs/src/pages/demos/snackbars/ConsecutiveSnackbars.tsx

```diff
@@ -94,8 +94,8 @@ function ConsecutiveSnackbars() {
 
   return (
     <div>
-      <Button onClick={e => handleClick('Message A')}>Show message A</Button>
-      <Button onClick={e => handleClick('Message B')}>Show message B</Button>
+      <Button onClick={e => handleClick(e, 'Message A')}>Show message A</Button>
+      <Button onClick={e => handleClick(e, 'Message B')}>Show message B</Button>
```

Comment:
This seemed fine before. Why did you need to change this?

Reply:
It's because ESLint failed the build because `e` was unused

Reply:
Then we can remove `e`.

Reply:
Then just don't use it i.e. () => `handleClick('Message A');`